lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -1241,7 +1241,23 @@ func (c *ContainerProxy) Remove(vc *viccontainer.VicContainer, config *types.Con
 // once the said container has been removed. It fetches a list of volumes that are joined
 // to at least one other container, and calls the portlayer to remove this container's
 // anonymous volumes if they are dangling. Errors, if any, are only logged.
-func removeAnonContainerVols(pl *client.PortLayer, cID string, volumes map[string]struct{}) {
+func removeAnonContainerVols(pl *client.PortLayer, cID string, vc *viccontainer.VicContainer) {
+	// NOTE: these strings come in the form of <volume id>:<destination>:<volume options>
+	volumes := vc.Config.Volumes
+	// NOTE: these strings come in the form of <volume id>:<destination path>
+	namedVolumes := vc.HostConfig.Binds
+
+	// assemble white list of volume paths before processing binds. MUST be paths, as we want to move to honoring the proper metadata in the "volumes" section in the future.
+	whitelist := make(map[string]struct{}, 0)
```

Comment:
given this is `removeAnonContainerVols` a _whitelist_ to me implies those that will be deleted.
`maskList` or `preserve` or similar would be a more explicit description of it's purpose.